IBM/carbon-addons-iot-react
# fix(table): reducer should not reset the page if the data changes

Closes:
Internal bug
https://github.ibm.com/wiotp/monitoring-dashboard/issues/769

**Summary**

- the bug was that the table page would reset if the data changes.  This causes problems if the user has made a paging decision and chooses to refresh the table data
This PR prereqs the th header wrap word-break:
https://github.com/IBM/carbon-addons-iot-react/pull/941

**Change List (commits, features, bugs, etc)**
- fix(tableReducer): use the page size from state, don't reset to the original every time the TABLE reregisters its data
- test(sidenav): added a page title bar story to the sidebar
- refactor(Table): remove the styled components from the base table and replace with scss as it was causing a strange failure to hide the pagination dropdown
- fix(TableCard): prevent whitespace warp on the table card because we need to dynamically size the height and assume a given row size to measure

**Acceptance Test (how to verify the PR)**

- Verify the table card stories
- Verify the table stories
